trunk/SUAVE/Methods/Performance/estimate_landing_field_length.py

```diff
@@ -5,14 +5,16 @@
 # Modified: Apr 2015, M. Vegh 
 #           Jan 2016, E. Botero 
 #           Mar 2020, M. Clarke
+#           Jul 2020, E. Botero 
 
 # ----------------------------------------------------------------------
 #  Imports
 # ----------------------------------------------------------------------
 
 import SUAVE
-from   SUAVE.Core            import Data
-from   SUAVE.Core            import Units
+from   SUAVE.Core import Data
```

Comment:
Formatting looks odd here

Reply:
I think I fixed this?

Reply:
Lining up SUAVE with the import and from seems odd to me, though I don't think we have any hard rules for it.

Reply:
oh lol, I didn't put that like that in the first place. I'm obviously a hard proponent of style, but I believe it should be enforced to ensure readability. I don't think this hurts or helps.